### ISSUE TITLE
prec / precision asymmetry

### ISSUE THREAD

**Issue by saschatimme** [kalmarek/Arblib.jl#34]
This annoys me on a purely aesthetic level:
```julia
a = Arb(2; prec = 128)
b = Arb(prec = 2 * precision(a))
```
This should either be both `prec` or both `precision`. I know there is `precision` in `Base` but I think nobody writes generic code using `precision` anyways.

**Comment by Joel-Dahne** [comment]
That's true! Maybe there's not many people using `precision` in base now, but I really like that they have it and wouldn't want to move away from it. One solution could be to allow both `prec` and `precision` as keyword arguments? Not sure if there is a good way to do that. 

From Julia 1.5 there is also some reason to want them to be different. The you could do something like
```
prec = 64
for i in 1:N
    Arb(2; prec)
    # Computations...
    prec *= 2
end
```
If they where called the same thing you would overwrite the `precision` method with the `precision` variable.

**Comment by Joel-Dahne** [comment]
The naming still annoys me slightly as well, but I think we are stuck with it at this point 😃
